editor/src/core/shared/uid-utils.ts

```diff
@@ -20,6 +20,67 @@ import { objectMap } from './object-utils'
 
 export const UtopiaIDPropertyPath = PP.create(['data-uid'])
 
+const atoz = [
+  'a',
+  'b',
+  'c',
+  'd',
+  'e',
+  'f',
+  'g',
+  'h',
+  'i',
+  'j',
+  'k',
+  'l',
+  'm',
+  'n',
+  'o',
+  'p',
+  'q',
+  'r',
+  's',
+  't',
+  'u',
+  'v',
+  'w',
+  'x',
+  'y',
+  'z',
+]
+
+// Assumes possibleStartingValue consists only of characters that are valid to begin with.
+export function generateConsistentUID(
```

Comment:
Should we completely replace `generateUID` with this now? Or is there a reason to keep the old one about?

Reply:
I think it still potentially makes sense to keep that for things like action changes, as that's got no consistency benefits/requirements. At least for now.